migrations/versions/57e3e1d9716a_initial_migration.py

```diff
@@ -0,0 +1,46 @@
+"""Initial migration
+
+Revision ID: 57e3e1d9716a
+Revises: 
+Create Date: 2022-12-21 13:35:54.026105
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '57e3e1d9716a'
+down_revision = None
+branch_labels = None
+depends_on = None
+
+
+def upgrade() -> None:
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table('zone',
+    sa.Column('id', sa.Integer(), nullable=False),
+    sa.Column('name', sa.String(), nullable=False),
+    sa.Column('is_online', sa.Boolean(), nullable=False),
```

Comment:
Rule of thumb — better to skip those prefixes like `has_`, `is_` or similar. `online: true|false` would give you same understanding but better readability.

Reply:
thnx!